src/main/resources/templates/javascriptMethods.js

```diff
@@ -0,0 +1,141 @@
+<script> 

```

Comment:
Give this a more descriptive name than "javascriptMethods.js".  The ".js" extension already indicate it's JavaScript methods... what methods are these?